assets/prototype/application/valueObjects/config/DateTimeFormats.ts

```diff
@@ -0,0 +1,19 @@
+import { defaultDateFormat, defaultTimeFormat } from './defaultDateTimeFormats';
+
+export type DateTimeFormatsProps = {
+    dateFormat: string;
+    timeFormat: string;
+    dateTimeFormat?: string;
+}
+
+export const DateTimeFormats = (formats: DateTimeFormatsProps): DateTimeFormatsProps => {
+    const maybeDefault = {
```

Comment:
This reminds me of monads :+1: 
Didn't used, just read something about that.
https://en.wikipedia.org/wiki/Monad_(functional_programming)#An_example:_Maybe